guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+> [we'd like to hear from you](https://coder.com/contact).
+
+|                   | Coder v1 | Coder OSS                                                                                       |
+| ----------------- | -------- | ----------------------------------------------------------------------------------------------- |
+| Avatar            | ✅       | ❌                                                                                              |
+| Dotfiles          | ✅       | Per-workspace [(dotfiles docs)](https://coder.com/docs/coder-oss/latest/dotfiles)               |
+| Generated SSH key | ✅       | ✅                                                                                              |
+| Default shell     | ✅       | Per-workspace [(with parameters)](https://coder.com/docs/coder-oss/latest/templates#parameters) |
+| Auto-start times  | ✅       | Per-workspace                                                                                   |
+| Git OAuth         | ✅       | SSH key only                                                                                    |
+
+<small>Something missing, or have feedback?
+[Let us know](https://coder.com/contact)</small>
+
+User-wide settings (e.g. avatar, shell, autostart times, dotfiles URL) are not
+currently supported in Coder OSS
+[(#3506)](https://github.com/coder/coder/issues/3506).
+
+### Workspaces
+
+To migrate Coder v1 workspaces, you'll need at least one
+[template](https://coder.com/docs/coder-oss/latest/templates) in your Coder OSS
+deployment, specifically with the image(s) you support in Coder v1.
+
+> If you are interested in a bulk user and/or workspace migration utility,
+> [we'd like to hear from you](https://coder.com/contact).
+
+From there, we recommend manually creating a workspace in Coder OSS and using a
+utility such as `scp` or `rsync` to copy the home directory from your v1
+workspace.
+
+```sh
+# Inside a Coder v1 workspace terminal (e.g. coder-v1.example.com)
+
+# Download the Coder OSS CLI (alias "codeross")
+curl -sL https://coder.com/install.sh | sh -s -- --method=standalone --binary-name=codeross > /dev/null
+
+# Log in to the Coder OSS deployment (e.g. coder-v2.example.com)
+codeross login https://coder-v2.example.com
+
+# Create a workspace
+codeross create <workspace-name>
+
+# Gain SSH access to v2 workspaces
+codeross config-ssh -y
+
+# Copy your home directory into the new Coder OSS workspace
+rsync \
+    --recursive \
+    --itemize-changes \
+    --info=progress2 \
+    --links \
+    --exclude='.cache/' \
+    $HOME/. coder.$CODER_WORKSPACE_NAME:/home/coder/.
+```
+
+|                                                                      | Coder v1                                                                                                          | Coder OSS                                                                                                            |
```

Comment:
what is this doing here?? this chart jsut comes out of no where/.... context please